### PR TITLE
update aggregate tooltip

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
The current tooltip for the aggregate toggle is not accurate.

## Description
This PR updates the english aggregate toggle tooltip to describe what the toggle does.

## Help wanted
I would be great to get some help with translations for:

- [x] Spanish
- [x] French
- [ ] Portuguese
- [ ] Slovenian
- [x] Swedish 
- [x] German

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
I'll take care of the Swedish translations.

**Comment by VIKTORVAV99** [comment]
@tonypls do you want me to update the swedish translation directly in this pr?

**Comment by tonypls** [comment]
> @tonypls do you want me to update the swedish translation directly in this pr?

Yeah of course :D

**Comment by VIKTORVAV99** [comment]
> > @tonypls do you want me to update the swedish translation directly in this pr?
> 
> Yeah of course :D

I'll do it right away then!

File: web/public/locales/sv.json [hunk 1]
```diff
@@ -168,7 +168,7 @@
     "dataIsDelayed": "Data är fördröjd",
     "aggregated": "aggregerad",
     "detailed": "detaljerad",
-    "aggregateinfo": "Länder med zoner som sakanar data kan inte bli aggregerade"
+    "aggregateinfo": "Växla mellan en detaljerad zon vy och en aggregerad lands vy"
```
Comment (VIKTORVAV99) [inline web/public/locales/sv.json:171]:
```suggestion
    "aggregateInfo": "Växla mellan en detaljerad zon vy och en aggregerad lands vy"
```

**Review by madsnedergaard — APPROVED** [review]

**Review by VIKTORVAV99 — COMMENTED** [review]
There is one place we need to switch to camel case. I'll approve after that.

**Comment by VIKTORVAV99** [comment]
And the automerge strikes again...

File: web/public/locales/sv.json (hunk 1 above)
Comment (madsnedergaard) [inline web/public/locales/sv.json:171]: This one has incorrect casing 😅

File: web/public/locales/es.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
     "dataIsEstimated": "Algunos de los datos de esta región son estimados. Lee más sobre la metodología de estimación <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Estimation-methods\" target=\"_blank\">aquí</a>.",
     "exchangesAreMissing": "Los intercambios son considerados en los cálculos pero no son mostrados actualmente. Lee más acerca de nuestros agregados históricos <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Historical-aggregates\" target=\"_blank\">aquí</a>.",
     "estimatedTooltip": "Los datos en tiempo real para esta zona son estimados basados en datos de las fuentes y suposiciones del modelo.",
-    "aggregatedTooltip": "Los datos para esta zona son agregados basados en valores horarios históricos",
+    "aggregatedTooltip": "Alternar entre visualización a nivel de zona o vista agregada de país",
```
Comment (madsnedergaard) [inline web/public/locales/es.json:64]: This translation is not correct - but I also don't think this label is used for anything? 

English version is this:
    "aggregatedTooltip": "The data for this zone is aggregated based on historical hourly values",

**Comment by madsnedergaard** [comment]
Sorry, I was too fast on approving 😬

File: web/public/locales/es.json (hunk 2 above)
Comment (VIKTORVAV99) [reply web/public/locales/es.json:64]: Didn't we use it for this label before?
image
(the gray aggregated one)

Comment (tonypls) [reply web/public/locales/es.json:64]: Oh, I've updated the wrong translation 🤦

**Pull request merged**